### ISSUE TITLE
Updating en docs

### ISSUE THREAD

**Issue by minggo** [cocos2d/cocos2d-x-docs#8]
## Getting Started

- [x] About Cocos2d-x
- [ ] Why Cocos2d-x
- [ ] How to learn
- [ ] Where to get help
- [ ] How to contribute

....

**Comment by slackmoehrle** [comment]
@minggo I tried to make a format that allows us to see both English and Chinese progress but I could not get the __to-do__ functionality to work inside a table. Would you like me to re-do this?

**Comment by minggo** [comment]
> but I could not get the to-do functionality to work inside a table. Would you like me to re-do this?

what does it mean?

**Comment by slackmoehrle** [comment]
it means I could not make check boxes like you had to just check off when complete.

**Comment by minggo** [comment]
Oh, not problem, this format works too.

**Comment by slackmoehrle** [comment]
for installation docs, I need to test 3.16 and install instructions on platforms right, just in case? OS X, Linux, Windows, etc? @drelaptop says he has tested Android already. 对不对？

**Comment by minggo** [comment]
I think these sections are not needed:

* VR: we don't maintain it now, not device to test too
* Tizen, Windows Phone: they are not supported now

**Comment by slackmoehrle** [comment]
Can we keep them and mark them deprecated? Just in case someone still uses them? We can mark the docs as 3.15 only?

**Comment by minggo** [comment]
yep, mark them as deprecated is a good idea, and `Android - Eclipse` should be deprecated too.

**Comment by slackmoehrle** [comment]
知道了

**Comment by minggo** [comment]
Tizen and Windows Phone are not supported in v3.16 too

**Comment by slackmoehrle** [comment]
yup. I will mark deprecated and only for v3.15 as last supported version for Tizen, Windows Phone and Eclipse.

**Comment by drelaptop** [comment]
@slackmoehrle For installation docs I have tested on Android Studio 2.3.3 , Visual Studio 2015, IOS on Xcode8.3.3, macOS on XCode8.3.3, cpp-tests project works well. can you tested it on Linux and Android-Command Line Tool ？

**Comment by slackmoehrle** [comment]
Priorities:
- [] Creatot To Cocos2d-x
- [] SQLite
- [] Multi-resolution
- [] Audio Engine
- [] Prebuilts
- [] Test installation -> Windows, OS X, Linux and modify install docs if needed

**Comment by drelaptop** [comment]
__3 Chapters__ be left for translation works

**Comment by drelaptop** [comment]
Now, the Chinese docs keep the same `[in-progress]` with English docs except `[deprecated] ` contents. @minggo

**Comment by minggo** [comment]
@drelaptop thanks.
@slackmoehrle when will you finish the 3 docs? I will deploy it to Chinese forum when they are ready.

**Comment by slackmoehrle** [comment]
I have 4 docs to complete. I updated the list above. I am busy for a few days this week, but I will try and work on them.
